### PR TITLE
Update README, add contributing guidelines and project roadmap

### LINKED ISSUE #19: Update README
Update the project's README file. It should include the following:
- What is eyespeak and why was it created
- list of features & capabilities
- screenshots
- Supported iOS versions/devices
- How to build the project
- Plans for the future
- Information on how to contribute / code of conduct
- What license does the project use

Here are some open source iOS apps with good READMEs to use as an example:
- https://github.com/Dimillian/SwiftHN
- https://github.com/artsy/eidolon
- https://github.com/wordpress-mobile/WordPress-iOS
- https://github.com/mozilla-mobile/firefox-ios
- https://github.com/fullstackio/FlappySwift

### PR DESCRIPTION
**Pull request by pg8wood**
Here's a first pass at a more open-source-friendly README. There's a few more additions I'd like to run by the team:
- Adding a short architecture overview
- Is there value in adding building instructions / a note on how WillowTree owns the provisioning?
- Thoughts on adding something to help hype up any open-source contributions? I've seen  some open-source projects have success with something like [hall-of-fame](https://github.com/sourcerer-io/hall-of-fame): it automates celebrating contributions, and is a neat way to thank contributors that come our way.

Closes #19

### CODE REVIEW

**Review by stevefosterwta — APPROVED** [review]
👍

**Comment by pg8wood** [comment]
Protip use the `rich diff` button in code review to see how GitHub will render the markdown! 🎉
image

**Review by stevefosterwta — APPROVED** [review]
✍️ 👍

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Vocable Roadmap
+
+- [ ] Handset Support
```
Comment (jsmorgan42) [inline ROADMAP.md:3]: Can this be filled in since we're about to ship handset support?

**Review by jsmorgan42 — APPROVED** [review]
👨‍🏫

**Pull request merged**